src/main/webapp/pipe.js

```diff
@@ -1,6 +1,15 @@
 function pipelineUtils() {
     var self = this;
     this.updatePipelines = function(divNames, errorDiv, view, fullscreen, page, component, showChanges, aggregatedChangesGroupingPattern, timeout, pipelineid, jsplumb) {
+
+        // Don't refresh pipelines if the tab/window is not active
+        if (document.hidden) {
```

Comment:
Could you please implement same change for the **src/main/webapp/themes/overview/pipe.js**